pkgs/development/python-modules/kafka-python/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, tox }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  version = "1.3.5";
+  pname = "kafka-python";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19m9fdckxqngrgh0www7g8rgi7z0kq13wkhcqy1r8aa4sxad0f5j";
+  };
+
+  buildInputs = [ tox ];
```

Comment:
checkInputs

Reply:
If tests run successfully without any `checkInputs`, is there a need to add them?